GlimpseImputationPipeline/Glimpse2Imputation.wdl

```diff
@@ -325,7 +325,9 @@ import pandas as pd
 hl.init(default_reference='GRCh38', idempotent=True)
 vcf = hl.import_vcf('~{imputed_vcf}', force_bgz=True)
 qc = hl.sample_qc(vcf)
-qc.cols().flatten().rename({'sample_qc.' + col: col for col in list(qc['sample_qc'])}).export('~{output_basename}.qc_metrics.tsv')
+qc_pd = qc.cols().flatten().rename({'sample_qc.' + col: col for col in list(qc['sample_qc'])}).to_pandas()
+qc_pd = qc_pd.rename(columns={'s': 'sample_id'})
```

Comment:
why separate the renames between hail and pandas?  why not just do them all in one or the other?

Also why do you need to convert to pandas at all?  Is it to get the `float_format` option when writing out the csv?

Reply:
I changed the renaming. And yes, we need it to be in pandas because Hail outputs it in scientific e+00x notation, and Esme requested a 0.000 output.

These changes make the WDL test fail because the output format is different. I'm waiting for the workflow to finish with the latest changes, to then change the expected test outputs in the palantir-workflows-test-data bucket before the merge